include/clang/Interpreter/CppInterOp.h

```diff
@@ -439,6 +439,8 @@ namespace Cpp {
   /// GetResourceDir() function.
   void AddIncludePath(const char *dir);
 
+  std::vector <std::string> GetIncludePath(const char *dir);
```

Comment:
```suggestion
  void GetIncludePath(const char *dir, std::vector <std::string>& Paths);
```

We should use that signature as it is more efficient. Can you also add tests for this?